central/tlsconfig/tlsconfig.go

```diff
@@ -36,17 +38,47 @@ func GetAdditionalCAs() ([][]byte, error) {
 		return nil, errors.Wrap(err, "reading additional CAs directory")
 	}
 
-	var certDERs [][]byte
+	var files []string
 	for _, certFile := range certFileInfos {
-		if filepath.Ext(certFile.Name()) != ".crt" {
-			log.Infof("Skipping additional-ca file %q, must end with '*.crt'.", certFile.Name())
+		if certFile.IsDir() {
```

Comment:
It may not be possible to loop into a directory in a secret mount, but to be safe adding a log statement a directory was skipped is useful.